### ISSUE TITLE
Toast Position

### ISSUE THREAD

**Issue by supreettare** [aritchie/userdialogs#279]
1) Please check all of the platforms you are having the issue on
-  iOS
- PCL

2) Please state the issue to the best of your ability below
In a simple xamarin forms app with a single entry, for ios, the toast notification appears behind the keyboard when the entry is focused. 
Is there a way we can fix it? 
I notice I can't change the position of the toast to appear from top of the screen instead. 
Please do let me know if there is a way I can workaround this issue. 

P.S. (The event where the toast appears, I do not have access to the entry under consideration so can't do Entry.Unfocus() befor the toast is triggered.

**Comment by supreettare** [comment]
Thanks for considering this Allan, have been using this lib in all my projects & is a wonderful add on. I will keep an eye on it for when a fix is available. 

thanks

**Comment by EmmanVazz** [comment]
@aritchie I wanted to do this as well but I just used the TTGSnackbar project directly for it. I actually committed a pr [fix](https://github.com/MarcBruins/TTGSnackbar-Xamarin-iOS/pull/6) a while back, it was crashing when I set the position. I was thinking of eventually updating this repo but haven't looked at the other platform implementations yet. Also I am too busy hacking away at other things right now, just thought you might want to know that info if you experienced that issue before.
